woocommerce/woocommerce
# Variation name in add to cart message for variations

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Reopening PR https://github.com/woocommerce/woocommerce/pull/31472, because it was accidentally closed. This was listed in https://github.com/woocommerce/woocommerce/issues/28991 as a feature request, but I think it's really a bug, and the variation name should be displayed in the add to cart message instead of the parent. By using the variation_id instead of the product_id the correct name is displayed.
Fixes https://github.com/woocommerce/woocommerce/issues/28991



### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Add a product variation to the cart from the single product page
2. Check the displayed message that confirms the product was added to the cart
3. Confirm that the variation name is displayed instead of the product name



### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message Display variation name in add to cart message instead of parent product for variations

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
